inbm/dispatcher-agent/dispatcher/sota/sota.py

```diff
@@ -356,7 +357,7 @@ def execute_from_manifest(self,
                     self._update_logger.status = OTA_PENDING
                     self._update_logger.error = ""
                 self._update_logger.save_log()
-                if self.sota_mode == 'download-only' or self._device_reboot in ["No", "N", "n", "no", "NO"]:  # pragma: no cover
+                if self.sota_mode == 'download-only' or rebooter.is_reboot(self._device_reboot):
```

Comment:
The condition `rebooter.is_reboot(self._device_reboot)` seems to be inverted. It should not reboot when `device_reboot` is set to a variant of "no". [important]